jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -140,6 +140,9 @@ then
 elif [[ "${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_centos" ]]
 then
   make feature_tests
+elif [[ "${TESTS_FOR}" == "e2e_tests" ]]
+then
+  make test-e2e
```

Comment:
I think you need to cd/pushd to local CAPM3 path in order to call `test-e2e` Makefile target of CAPM3 repo. Because, at this point we are inside l[ocal copy of metal3-dev-env](https://github.com/metal3-io/project-infra/blob/master/jenkins/scripts/files/run_integration_tests.sh#L127-L134) repository.

Reply:
Good point, fixed